marc.py

```diff
@@ -0,0 +1,612 @@
+from nose.tools import set_trace
+import datetime
+from StringIO import StringIO
+from flask_babel import lazy_gettext as _
+import re
+
+from pymarc import (
+    Field,
+    Record,
+    MARCWriter
+)
+from config import (
+    Configuration,
+    CannotLoadConfiguration,
+)
+from model import (
+    get_one,
+    get_one_or_create,
+    BaseMaterializedWork,
+    CachedMARCFile,
+    Collection,
+    ConfigurationSetting,
+    DeliveryMechanism,
+    Edition,
+    ExternalIntegration,
+    Identifier,
+    Representation,
+    Session,
+    Work,
+)
+from classifier import Classifier
+from s3 import S3Uploader
+from lane import Lane
+
+class Annotator(object):
+    """The Annotator knows how to add information about a Work to
+    a MARC record."""
+
+    marc_cache_field = Work.marc_record.name
+
+    # From https://www.loc.gov/standards/valuelist/marctarget.html
+    AUDIENCE_TERMS = {
+        Classifier.AUDIENCE_CHILDREN: "Juvenile",
+        Classifier.AUDIENCE_YOUNG_ADULT: "Adolescent",
+        Classifier.AUDIENCE_ADULTS_ONLY: "Adult",
+        Classifier.AUDIENCE_ADULT: "General",
+    }
+
+    # TODO: Add remaining formats. Maybe there's a better place to
+    # store this so it's easier to keep up-to-date.
+    # There doesn't seem to be any particular vocabulary for this.
+    FORMAT_TERMS = {
+        (Representation.EPUB_MEDIA_TYPE, DeliveryMechanism.NO_DRM): "EPUB eBook",
+        (Representation.EPUB_MEDIA_TYPE, DeliveryMechanism.ADOBE_DRM): "Adobe EPUB eBook",
+        (Representation.PDF_MEDIA_TYPE, DeliveryMechanism.NO_DRM): "PDF eBook",
+        (Representation.PDF_MEDIA_TYPE, DeliveryMechanism.ADOBE_DRM): "Adobe PDF eBook",
```

Comment:
I don't know if this is where you got them but these look like the names used by Overdrive to describe formats.